doc/vxlan/EVPN/EVPN_VxLAN_Multihoming.md

```diff
@@ -0,0 +1,1671 @@
+ VTEPs: 2.2.2.2(EV)
+
+```
+
+**show bgp l2vpn evpn es-evi vni (1-16777215)**
+```
+sonic# do show bgp l2vpn evpn es-evi vni 100
+Flags: L local, R remote, I inconsistent
+VTEP-Flags: E EAD-per-ES, V EAD-per-EVI
+VNI      ESI                            Flags VTEPs
+100      00:11:22:33:44:55:66:77:88:01  LR    4.1.1.1(EV)
+100      01:b8:6a:97:e2:4c:9f:00:02:00  R     2.1.1.1(EV),3.1.1.1(EV)
+100      03:00:00:11:22:33:03:00:00:03  LR    2.1.1.1(EV),3.1.1.1(EV),4.1.1.1(EV)
+
+```
+
+**show bgp l2vpn evpn es-vrf detail**
+```
+sonic# show bgp l2vpn evpn es-vrf detail
+ES-VRF Flags: A Active
+ESI                            VRF             Flags IPv4-NHG IPv6-NHG Ref
+03:00:00:00:11:22:33:00:00:01  (null)          A     72580645 72580646 2
+```
+
+**show bgp l2vpn evpn next-hop**
+```
+sonic# show bgp l2vpn evpn next-hops
+VRF             IP              RMAC              #Paths     Base Path
+RED             10.10.10.3      44:38:39:22:01:bb 4          10.1.10.0/24
+RED             10.10.10.4      44:38:39:22:01:c1 4          10.1.10.0/24
+RED             10.10.10.2      44:38:39:22:01:af 4          10.1.10.0/24
+BLUE            10.10.10.3      44:38:39:22:01:bb 2          10.1.30.0/24
+BLUE            10.10.10.4      44:38:39:22:01:c1 2          10.1.30.0/24
+BLUE            10.10.10.2      44:38:39:22:01:af 2          10.1.30.0/24
+```
+
+<a id="353-Debug-Commands"></a>
+### 3.5.3 Debug Commands
+
+Following FRR debug commands are available for EVPN VxLAN and Mulithoming debugging:
+
+```
+sonic(config)# log syslog debugging
+
+sonic(config)# debug bgp evpn mh
+  es     Ethernet Segment debugging
+  route  Route debugging
+sonic(config)# debug bgp zebra
+
+sonic(config)# debug zebra evpn mh
+  es     Ethernet Segment Debugging
+  mac    MAC Debugging
+  neigh  Neigh Debugging
+  nh     Nexthop Debugging
+sonic(config)# debug zebra vxlan
+sonic(config)# debug zebra kernel
+sonic(config)# debug zebra dplane
+sonic(config)# debug zebra fpm
+
+```
+<a id="354-REST-API-Support"></a>
+### 3.5.4 REST API Support
+
+** Ethernet-segment configuration **
+```
+curl -X PATCH "https://SWITCH_IP:9090/restconf/data/openconfig-network-instance:network-instances/network-instance=default/evpn/ethernet-segments/ethernet-segment=PortChannel17" -H "accept: */*" -H "Content-Type: application/yang-data+json" -d "{\"openconfig-network-instance:ethernet-segment\":[{\"name\":\"PortChannel17\",\"config\":{\"esi-type\":\"TYPE_0_OPERATOR_CONFIGURED\",\"esi\":\"0017000000000000000a\"},\"df-election\":{\"config\":{\"preference\":201}}}]}"
+```
+
+** EVPN Multihoming global configuration **
+```
+curl -X PATCH "https://SWICH_IP:9090/restconf/data/openconfig-network-instance:network-instances/network-instance=default/evpn/evpn-mh/config" -H "accept: */*" -H "Content-Type: application/yang-data+json" -d "{\"openconfig-network-instance:config\":{\"startup-delay\":150,\"mac-holdtime\":200,\"neigh-holdtime\":250}}" 
+```
+<a id="4-Flow-Diagrams"></a>
+# 4 Flow Diagrams
```

Comment:
The Flow Diagrams section contains multiple distinctly different ways of to naming/implementing feature comparing to the rest of the document.  We think these diagrams are outdated artifacts from #1638 requirements and contradict the new requirements.

Reply:
hummm. Can you be explicit?